packages/react-scripts/scripts/build-module.js

```diff
@@ -1,58 +1,82 @@
 require('../utils/loadEnv');
 
-var paths = require('../config/paths');
+var path = require('path');
+var fs = require('fs-extra');
+var chokidar = require('chokidar');
+var transformFileSync = require('babel-core').transformFileSync;
 var spawn = require('cross-spawn');
-var result;
+var paths = require('../config/paths');
 
-var defaults = [
-  '--presets=trunkclub',
-  '--source-maps'
-];
+function lint () {
+  return spawn.sync('node', [require.resolve('./lint')], {stdio: 'inherit'});
+}
 
-var directoryArgs = [
-  paths.appSrc,
-  '--out-dir=' + paths.appBuild
-];
+function transformWithBabel (filePath) {
+  const { code } = transformFileSync(path.join(paths.appSrc, filePath), {
+    ast: false,
+    sourceMaps: 'inline',
```

Comment:
🤔 

I wasn't able to find a method in the babel node API that also wrote to disk; however, the `transformFileSync` method _does_ return the source map. I can write this to disk too but I'll have to find a way to get the `sourceMapping` comment at the end of the generated code.

As far as where these are loaded, being module output, the only thing consuming the source map would be webpack (or node). The `source-map-loader` can parse the data URIs used for inline styles and would create external source maps when our apps are built. I hope that makes sense. I is a bit tricky to describe but there are several layers of parsing and source maps that are happening for our code.